src/ci/scripts/upload-artifacts.sh

```diff
@@ -23,6 +23,9 @@ fi
 # CPU usage statistics.
 cp cpu-usage.csv "${upload_dir}/cpu-${CI_JOB_NAME}.csv"
 
+# Build metrics generated by x.py.
+cp "${dist_dir}/../build-metrics.json" "${upload_dir}/build-metrics--${CI_JOB_NAME}.json"
```

Comment:
Why two dashes in between build-metrics and the ci job name?

Reply:
Mostly to have a clearer separator, and following [your comment on Zulip](https://rust-lang.zulipchat.com/#narrow/stream/242791-t-infra/topic/build.20timings.20from.20x.2Epy/near/270910539). I can change it either way though.

Reply:
Heh, I think that was just a typo. I don't see much benefit to a double dash. Either way is probably fine but let's prefer a single dash since it's easy to change now.

Reply:
Fixed.